README.md

```diff
@@ -27,9 +27,9 @@ Revolver currently supports the following AWS resources:
  * EC2 instances including those run in Autoscaling, Revolver will pause the ASG prior to shutting down instances
  * EBS volumes. Revolver will tag these resources. Revolver can get the tags from the parent instance/volume.
  * Snapshots. Revolver will tag these resources. Revolver can get the tags from the parent instance/volume.
- * RDS single instances, Revolver will use the native start/stop featur
- * RDS multi-az instances, Revolver will snapshot the instance and delete it. In the morning it will be restored from snapshot
- * RDS Clusters, Revolver will snapshot the cluster and save the critical information about the cluster members on the snapshot
+ * RDS single instances, Revolver will use the native start/stop feature
+ * RDS multi-az instances, Revolver will use the native start/stop feature
+ * RDS Clusters, Revolver will use the native start/stop feature
```

Comment:
Do you need to explicitly say the mechanism, now that all are native?

Reply:
explicit is better than implicit. also you can track on the diff what the changes are.